common/models/model-definition.js

```diff
@@ -136,7 +136,9 @@ function ready(ModelDefinition) {
         async.parallel([
           removeModelDef,
           removeModelDefJs
-        ], cb);
+        ], function(results) {
```

Comment:
`async.parallel` returns an array and the order depends on which task finish 1st. I wonder if your change will break `strong-arc`.

@superkhau I think you should work with @jtary to make sure the callback returns the same result before your PR.